backend/src/test/java/gov/cdc/usds/simplereport/integration/UploadTestResultsIntegrationTest.java

```diff
@@ -89,9 +89,7 @@ void setup() throws IOException {
 
     // gets auth token
     stubFor(
-        WireMock.post(
-                urlEqualTo(
-                    "/api/token?scope=simple_report.%2A.report&grant_type=client_credentials&client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3Aclient-assertion-type%3Ajwt-bearer&client_assertion"))
+        WireMock.post(urlEqualTo("/api/token"))
```

Comment:
To firm up the testing could we also assert the body of the post looks like what we expect? `.withRequestBody(equalTo("scope=simple_report.%2A.report&grant_type=client_credentials&client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3Aclient-assertion-type%3Ajwt-bearer&client_assertion"))`?